lib/tasks/scripts/link_teleconsultation_medical_officers.rb

```diff
@@ -0,0 +1,37 @@
+class LinkTeleconsultationMedicalOfficers
+  class << self
+    def call
+      linked_users_count, unlinked_numbers_count = 0, 0
+
+      Facility.all.each do |facility|
```

Comment:
Use `Facility.find_each` to avoid instantiating a bajillion Ruby objects to save memory!